Lib/posixpath.py

```diff
@@ -401,8 +409,18 @@ def _joinrealpath(path, rest, seen):
         pardir = '..'
 
     if isabs(rest):
-        rest = rest[1:]
-        path = sep
+        if not _vxworks:
+            rest = rest[1:]
+            path = sep
+        else:
+            if rest.startswith(sep):
+                rest = rest[1:]
+                path = sep
+            else:
+                ind = rest.find(':')
+                ind = 0 if ind < 0 else ind
+                path = rest[:ind]
+                rest = rest[ind:]
```

Comment:
Again, runtime testing of _vxworks.  Better to define a special _joinrealpath() function at module import time.